### PR TITLE
[8.0][MIG] quality_control_mrp_operations

### PR DESCRIPTION
**Pull request by mikevhe18**

### CODE REVIEW

File: quality_control_mrp_operations/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+===================================
+
+This module enables quality control for MRP operations.
+
+Installation
+============
+
+To install this module, you need to:
+
+1.  Clone the branch 8.0 of the repository https://github.com/OCA/manufacture
```
Comment (elicoidal) [inline quality_control_mrp_operations/README.rst:16]: not blocking but better to use "#." for enumerated lists

File: quality_control_mrp_operations/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+   :target: https://runbot.odoo-community.org/runbot/129/8.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/manufacture/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
```
Comment (elicoidal) [inline quality_control_mrp_operations/README.rst:36]: you can simplify the message following the newest template

File: quality_control_mrp_operations/models/mrp.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    qtemplate_id = fields.Many2one('qc.test', string='Test')
+
+    @api.model
+    def create(self, data):
+        if 'required_test' in data:
+            required_test = data.get('required_test')
+            if required_test:
+                if 'qtemplate_id' not in data:
+                    raise except_orm(_('Operation Creation Error!'),
+                                     _("You must define the test template"))
```
Comment (elicoidal) [inline quality_control_mrp_operations/models/mrp.py:22]: period at the end of the sentences

File: quality_control_mrp_operations/models/mrp.py [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                else:
+                    qtemplate_id = data.get('required_test')
+                    if not qtemplate_id:
+                        raise except_orm(_('Operation Creation Error!'),
+                                         _("You must define template test"))
```
Comment (elicoidal) [inline quality_control_mrp_operations/models/mrp.py:27]: ...define the template...

File: quality_control_mrp_operations/models/mrp.py [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+            required_test = vals.get('required_test')
+            if required_test:
+                if 'qtemplate_id' not in vals:
+                    raise except_orm(_('Operation Modification Error!'),
+                                     _("You must define the test template"))
+                else:
+                    qtemplate_id = vals.get('required_test')
+                    if not qtemplate_id:
+                        raise except_orm(_('Operation Modification Error!'),
+                                         _("You must define template test"))
```
Comment (elicoidal) [inline quality_control_mrp_operations/models/mrp.py:44]: idem

File: quality_control_mrp_operations/models/mrp.py (hunk 4 above)
Comment (elicoidal) [inline quality_control_mrp_operations/models/mrp.py:27]: isnot there any way to group those 2 tests together

Comment (elicoidal) [inline quality_control_mrp_operations/models/mrp.py:27]: Actually I am confused between the "template test" and the "test template"

File: quality_control_mrp_operations/models/mrp.py [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+    @api.model
+    def action_done(self):
+        if self.test_ids:
+            for test in self.test_ids:
+                if test.state not in ('success', 'failed', 'canceled'):
+                    raise except_orm(_('Finalization Operation Error!'),
+                                     _("There are quality tests in draft or"
+                                       " approval pending state for this"
+                                       " operation. Please finish or cancel "
+                                       "them."))
```
Comment (elicoidal) [inline quality_control_mrp_operations/models/mrp.py:150]: them before proceeding.

**Comment by LoisRForgeFlow** [comment]
Reviews unanswered for a long time. Closing, feel free to reopen.

**Pull request closed without merging**